meilisearch-release-changelog/v1.8.0.md

```diff
@@ -0,0 +1,219 @@
+# v1.8.0 release changelogs
+
+<!-- The following line should ONLY be put PRE-release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+Meilisearch v1.8 introduces new changes and optimizations related to the Hybrid search with the addition of new models and embedders like REST embedders and the Ollama model. This version also focuses on stability by adding more security around the search requests. Finally, we introduce the negative operator to exclude specific terms from a search query.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+Some SDKs might not include all new features—consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and improvements 🔥
+
+## Hybrid search improvements
+
+Full description of hybrid search changes [here](https://meilisearch.notion.site/v1-8-AI-search-API-usage-135552d6e85a4a52bc7109be82aeca42).
```

Comment:
Yes please, the changelog is not exhausitve about examples and usage, I would like this link to appear 😊

Reply:
Updated my suggestion. When I'm back we could perhaps have a chat about what information the usage document has that is missing from the docs. I think we'd ideally always direct users to the docs instead of the PRDs.

Reply:
If the docs are fully exhaustive and cover the full feature, I'm ok to not link the PRD 😊 
Is it the case for hybrid search?